pipelines/shopgate-project.smile-io.getSmileWaysToEarn.v1.json

```diff
@@ -0,0 +1,38 @@
+{
+  "version": "1",
+  "pipeline": {
+    "id": "shopgate-project.smile-io.getSmileWaysToEarn.v1",
+    "public": true,
+    "input": [
+      {"key": "sgxsMeta", "id": "100"},
+      {"key": "forceRefresh", "id": "150", "optional": true}
```

Comment:
@aaron-martin do you have some input for this one?

Reply:
This isn't used some where right now. However, I think it is possible it will be needed in the future. I think it will not hurt anything to leave it in or the time being.